Source/Controllers/MainViewControllers/ConnectionView/SPConnectionController.m

```diff
@@ -303,7 +303,7 @@ - (IBAction)initiateConnection:(id)sender
 	// for increased security.
 	if (connectionKeychainItemName && !isTestingConnection) {
 		if ([[keychain getPasswordForName:connectionKeychainItemName account:connectionKeychainItemAccount] isEqualToString:[self password]]) {
-			[self setPassword:[[NSString string] stringByPaddingToLength:[[self password] length] withString:@"sp" startingAtIndex:0]];
+			[self setPassword:@"SequelAceSecretPassword"];
```

Comment:
This is likely okay. Personally, I like to be able to tell the relative length of the password, but this is probably more secure than displaying the length of the password.